### PR TITLE
Set up infrastructure for client development

### PR DESCRIPTION
**Pull request by elemoine**
This PR sets up the infrastructure for client-side development. It also creates a simple Bootstrap-based index page with a typeahead-based search field.

Please review.

### CODE REVIEW

**Comment by ingenieroariel** [comment]
LGTM :+1:

File: development.ini [hunk 1]
```diff
@@ -17,6 +17,11 @@ pyramid.includes =
     pyramid_tm
 
 sqlalchemy.url = sqlite:///%(here)s/thinkhazard.sqlite
+
+jinja2.filters =
+    route_url = pyramid_jinja2.filters.route_url_filter
+    static_url = pyramid_jinja2.filters:static_url_filter
```
Comment (elemoine) [inline development.ini]: One of two filters is not correct.

File: package.json [hunk 2]
```diff
@@ -17,7 +17,13 @@
   },
   "homepage": "https://github.com/GFDRR/thinkhazard",
   "dependencies": {
+    "bootlint": "^0.11.0",
```
Comment (fvanderbiest) [inline package.json]: We should stick to one strategy: either set the versions or use the latest available.

File: development.ini [hunk 3]
```diff
@@ -17,6 +17,11 @@ pyramid.includes =
     pyramid_tm
 
 sqlalchemy.url = sqlite:///%(here)s/thinkhazard.sqlite
+
+jinja2.filters =
+    route_url = pyramid_jinja2.filters.route_url_filter
```
Comment (fvanderbiest) [inline development.ini]: Issue here

**Comment by fvanderbiest** [comment]
LGTM for merging when both comments are addressed.

**Comment by elemoine** [comment]
Thanks @ingenieroariel and @fvanderbiest for the reviews. I'll merge as soon as soon as Travis is happy.

**Pull request merged**